rpcd/playbooks/roles/filebeat/defaults/main.yml

```diff
@@ -58,6 +58,11 @@ multiline_openstack_pattern: '^[0-9-]{10} +[0-9:\.]+ +[0-9]+ +[A-Z]+ +[A-Za-z0-9
 #                                  Date        Time      Date         Time     PID     Level   Python Module   Request Id
 multiline_keystone_error_pattern: '[0-9-]{10} +[0-9:\.]+ [0-9-]{10} +[0-9:\.]+ [0-9]+ +[A-Z]+ +[A-Za-z0-9\._]+ \[|Traceback'
 
+multiline_swift_container_replicator_pattern: '^[A-Za-z]+[[:space:]]+ +[0-9]{1,2} +[0-9:\.]+ +[A-Za-z0-9-]+ container-replicator: +[A-Za-z0-9-\ ]+'
```

Comment:
For each of the other patterns, we include a play.golang.org link so people can look at the test data and see how modifying the regular expression would work. Can you include one here?